env.example

```diff
@@ -8,7 +8,7 @@
 
 IRC_BLACKLIST=""
 IRC_BOT_NAME=teleirc
-IRC_CHANNEL=#channel
+IRC_CHANNEL="#channel"
```

Comment:
@jwflory @Zedjones quotations needed to be added here in the `env.example` because the config parsing sees the `#` as a comment, therefore ignoring the IRC channel's name.

Reply:
LGTM